integration/test_ei_event

```diff
@@ -38,5 +38,5 @@ event_test() {
     "$(yes "completed" | head -n $expected_job_runs)"
 }
 
-# Arguments: config_name, address, account, expected_job_runs
-event_test "$1" "$2" "$3" "$4"
+# Arguments: config_name, address, account, expected_job_runs, function_selector, ABI
+event_test "$1" "$2" "$3" "$4" "$5" "$6"
```

Comment:
We could start putting all params in a JSON file, and just include that instead of appending to a never-ending line here. Something we can track in a separate story, or what do you think?